src/cleanvision/imagelab.py

```diff
@@ -21,29 +24,82 @@
 from cleanvision.utils.utils import get_filepaths, deep_update_dict
 from cleanvision.utils.viz_manager import VizManager
 
+__all__ = ["Imagelab"]
+
 OBJECT_FILENAME = "imagelab.pkl"
 TImagelab = TypeVar("TImagelab", bound="Imagelab")
 
 
 class Imagelab:
+    """A single class to find all types of issues in image datasets.
+
+    Parameters
+    ----------
+    data_path : str
+        Path to image files. Imagelab will recursively retrieve all image files from the specified path
+
```

Comment:
Todo: add optional `file_paths` arg

Reply:
Added

Reply:
@sanjanag Can we raise ValueError if both data_path and file_paths are provided?  And if they are both None

Seems like user messed something up in that case.